### PR TITLE
README: Document geth releases

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by lrettig** [comment]
Hi @chfast, thanks for helping improve this doc. To me, the format of this doc is a simple step by step "to do" checklist for people who want to connect to the testnet but don't necessarily know anything about geth or the other tooling. It seems like this change isn't a clear list of steps, but rather some information on the current state of EVMC support in geth, so I wonder if this is the best place for this information to live. Maybe we should put it into a separate doc?

**Comment by chfast** [comment]
My goal is to have a precise information how to build geth for testnet and what version is deployed on the testnet. It does not fit EVMC completely because there are additional changes added to geth that are not EVMC related.

**Comment by lrettig** [comment]
It's not clear to me from this how I would build geth for testnet. It's a list of branches/PRs, not a list of steps. Sorry if I'm missing something obvious!

File: README.md [hunk 1]
```diff
@@ -85,25 +85,37 @@ This section describes how to run geth node with ewasm backend.
...
+5. Allow setting EVMC by env var (TBD, workaround)
+6. acct-range-at (TBD)
+
+The pack of above changes are released regurally as git tag `ewasm-testnet-vX`:
+
+- TBD: [`ewasm-testnet-v1`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
+
+Deployed:
+
+- Production: TBD: [`ewasm-testnet-production`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
```
Comment (hugo-dc) [inline README.md]: Maybe we could clarify that right now we can use the same go-ethereum branch for both Production and staging.

File: README.md [hunk 2]
```diff
@@ -85,25 +85,37 @@ This section describes how to run geth node with ewasm backend.
...
+- Staging: TBD: [`ewasm-testnet-staging`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
+
+
+#### Building
+
+1. Build geth
 
-	Checkout the `evmc` branch of [this geth fork](https://github.com/chfast/go-ethereum/tree/evmc) ([PR](https://github.com/ethereum/go-ethereum/pull/17050)):
```
Comment (hugo-dc) [inline README.md]: Here we could specify again that we need to get the `ewasm-testnet-v1` branch just to be explicit in the building process.

**Comment by lrettig** [comment]
What's the most recent "bundle"? Is it https://github.com/ewasm/go-ethereum/releases/tag/ewasm-v3?

**Comment by chfast** [comment]
Yes.

**Comment by chfast** [comment]
I don't want to list the features we added. Is there any valuable information here you want to keep?

**Comment by chfast** [comment]
Review this again.

File: README.md [hunk 3]
```diff
@@ -100,11 +100,16 @@ Aleth supports EVMC out of the box, but geth not yet.
 
 1. Build geth with EVMC
 
-	Checkout the `evmc` branch of [this geth fork](https://github.com/chfast/go-ethereum/tree/evmc) ([PR](https://github.com/ethereum/go-ethereum/pull/17050)):
+	Checkout the `ewasm` branch of [the ewasm geth fork](https://github.com/ewasm/go-ethereum):
```
Comment (axic) [inline README.md]: No need to state `ewasm` branch. That is the master, just say clone the ewasm geth fork.

File: README.md [hunk 4]
```diff
@@ -100,11 +100,16 @@ Aleth supports EVMC out of the box, but geth not yet.
...
+	
+	Geth for ewasm testnet is released regurally as git tag `ewasm-testnet-milestoneX`:
```
Comment (axic) [inline README.md]: Regurally -> regularly

File: README.md [hunk 5]
```diff
@@ -137,7 +138,9 @@ Aleth supports EVMC out of the box, but geth not yet.
 
 1. Run geth with hera and connect to the testnet:
 
-	`geth` will check the `EVMC_PATH` environment variable for path to EVMC VM shared library. Point it to the hera shared library that you built a moment ago. Additionally `geth` will check the `EVMC_OPTIONS` environment variable for EVMC options (which are documented in [hera](https://github.com/ewasm/hera)). Multiple options can be specified by separating them with a space, e.g., `EVMC_OPTIONS='metering=true fallback=true'`.
+	Use `--vm.ewasm` flag in `geth` to plug in an EVMC VM shared library. Point it to the hera shared library that you built a moment ago.
```
Comment (axic) [inline README.md]: Hera should be capitalised here

File: README.md [hunk 6]
```diff
@@ -137,7 +138,9 @@ Aleth supports EVMC out of the box, but geth not yet.
 
 1. Run geth with hera and connect to the testnet:
 
-	`geth` will check the `EVMC_PATH` environment variable for path to EVMC VM shared library. Point it to the hera shared library that you built a moment ago. Additionally `geth` will check the `EVMC_OPTIONS` environment variable for EVMC options (which are documented in [hera](https://github.com/ewasm/hera)). Multiple options can be specified by separating them with a space, e.g., `EVMC_OPTIONS='metering=true fallback=true'`.
+	Use `--vm.ewasm` flag in `geth` to plug in an EVMC VM shared library. Point it to the hera shared library that you built a moment ago. 
+	Additional EVMC options can be provided after a comma. 
+	Hera options are documented in [hera](https://github.com/ewasm/hera)).
```
Comment (axic) [inline README.md]: Excess closing parenthesis

File: README.md [hunk 7]
```diff
@@ -100,23 +100,24 @@ Aleth supports EVMC out of the box, but geth not yet.
...
+
+	Geth for ewasm testnet is released regularly by tagging the above fork with `ewasm-testnet-milestoneX`, where
+	`X` stands for a milestone number.
+
+	(Latest: [`ewasm-testnet-milestone3`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-milestone3))
+
 	```sh
-	> git clone https://github.com/chfast/go-ethereum && cd go-ethereum
-	> git checkout evmc
+	> git clone https://github.com/ewasm/go-ethereum -b ewasm-testnet-milestone3
```
Comment (axic) [inline README.md]: So is it milestone3 or milestone1?

Comment (lrettig) [reply README.md]: Up to you, for some reason I thought I heard 3 on the call

Comment (axic) [reply README.md]: Okay changed to 1 for now.

**Pull request merged**